revolv/templates/project/project.html

```diff
@@ -17,53 +22,118 @@
-            <iframe width="420" height="315" id="project_video"></iframe>
-        {% endif %}
-        {% if project.categories|length > 0 %}
-            <h5>Categories</h5>
-            {% for category in project.categories %}
-                {{ category }}
-            {% endfor %}
-        {% endif %}
-        {% if project.donation_levels|length > 0 %}
-            <h5>Donation Levels</h5>
-            {% for donation in project.donation_levels %}
-                <p>Description: {{ donation.description }}, Amount: {{ donation.amount }}</p>
-            {% endfor %}
-        {% endif %}
-        {% if project.is_completed or project.is_active and project.post_funding_updates %}
-            <h5>Post Funding Updates</h5>
-            <p>{{ project.post_funding_updates}}</p>
-        {% endif %}
+
+<div class="row dashboard">
+    <div class="graphics-container">
+        <div class="top-bar">
+            <div class="container">
+                <p class="amount-donated">${{ project.amount_donated }} donated</p>
+                <p class="days-left">{{ project.days_left }} days left</p>
+            </div>
+        </div>
+        <div class="bottom-bar">
+            <div class="container">
+                <p class ="amount-saved">{{ project.actual_energy }} kwH saved</p>
+                <p class ="num-donors">{{ donor_count }} donors</p>
+            </div>
+        </div>
+        <svg class="svg-graphics-container">
+            <circle class="outside-circle" cx="50%" cy="50%" r="125" fill="white" />
+        </svg>
+        <svg class="internal-graphics-container"></svg>
+        <div class="text-container">
+            <p class="percentage-text">{{ project.percent_complete }}%</p>
+            <!-- <p class="percentage-text">70%</p> -->
+            <p class="funded">FUNDED</p>
+        </div>
+    </div>
+</div>
+
+<div class="heading row">
+    {% if project.cover_photo %}
+    <div class = "large-12 header-img-div" style='background-image: url("{{project.cover_photo.url}}")'>
+        <div class="overlay">
+            <div class="donate-button-header">
+               <a class="donate-link" href="#donate" data-reveal-id="donate-modal"><p>DONATE</p></a>
+            </div>
+            <p class ="project-title" >{{ project.title }}</p>
+        </div>
+    </div>
+    {% endif %}
+</div>
+
+
+<div class="row about">
+    <div class="header-text">
+        <p class="header">Promoting healthy business, healthy people, and a healthy planet</p>
+    </div>
+    <div class="gray-bar"></div>
+    <div class="row categories">
+        <div class="category-text"><p class="category-text-inside">CATEGORIES   </p></div>
+        {% for category in project.categories %}
+            <div class="category-holder">
+                <p class="category-label">{{ category }}</p>
+            </div>
+        {% endfor %}
+
+        <!-- These two are in here for testing and viewing purposes: projects don't have categories right now. -->
```

Comment:
oh what i meant was that this particular project didn't have categories attached to it specifically!